soh/src/overlays/actors/ovl_Item_Etcetera/z_item_etcetera.c

```diff
@@ -225,6 +225,10 @@ void ItemEtcetera_DrawThroughLens(Actor* thisx, GlobalContext* globalCtx) {
         func_8002ED80(&this->actor, globalCtx, 0);
 
         if(gSaveContext.n64ddFlag && globalCtx->sceneNum == 16) {
+            s32 randoGetItemId = GetChestGameRandoGetItemId(this->actor.room, this->giDrawId, globalCtx);
+            if (randoGetItemId >= GI_MINUET_OF_FOREST && randoGetItemId <= GI_DOUBLE_DEFENSE) {
+                EnItem00_CustomItemsParticles(&this->actor, globalCtx, randoGetItemId);
+            }
             GetItem_Draw(globalCtx, GetChestGameRandoGiDrawId(this->actor.room, this->giDrawId, globalCtx));
```

Comment:
no need to call the chestgamerandodrawid here now that we have the getitemid, we can just
```
GetItem_Draw(globalCtx, GetItemModelFromId(randoGetItemId));
```

Reply:
Done